go/libkb/env.go

```diff
@@ -1016,7 +1016,10 @@ func (e *Env) GetMountDir() (string, error) {
 		if runMode != ProductionRunMode {
 			return "", fmt.Errorf("KBFS is currently only supported in production mode on Windows")
 		}
-		return "k:", nil
+		return e.GetString(
```

Comment:
I thought so too at first, but then I thought, this is a KBFS setting, does it belong in keybase.exe? Wasn't sure how much those two should be mixed. I'll add the others.